pymc3/distributions/distribution.py

```diff
@@ -116,10 +136,14 @@ def __new__(cls, name, *args, **kwargs):
 
         # Some distributions do not accept shape=None
         if has_shape or shape is not None:
+            if "givens" in kwargs:
+                raise ValueError("givens found")
```

Comment:
This probably needs a more informative error message (you may already be working on that, but just wanted to make sure)

Reply:
this should be removed, they were debugging checkpoints :sweat_smile:

Reply:
Ah ok, that makes sense: I really didn't understand what they were doing 😅 
Ping me when you think this PR is ready for review then 😉